core/src/main/java/org/frankframework/core/MessageBrowserField.java

```diff
@@ -0,0 +1,18 @@
+/*
+   Copyright 2025 WeAreFrank
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.core;
+
+public record MessageBrowserField(String fieldName, String property, String displayName, String type) { }
```

Comment:
Het liefst zie ik hier nog een korte javadoc beschrijving. Nu moest ik zelf uitzoeken dat fieldName voor de database is, en ook alleen daar gebruikt wordt. En property is mij nog een beetje een raadsel, is dat de frontend naam?